### DISCUSSION TITLE
关于 Toolbar 组件开发相关

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#108] [Ideas]
# Toolbar

## 说明

Toolbar 工具栏分为两个组件 `f-toolbar` 和 `f-toolbar-item`

## 设计稿

https://www.figma.com/file/9wxCyf7hlkFgcK3ljwTfVs/Toolbar-%E5%B7%A5%E5%85%B7%E6%A0%8F?node-id=0%3A1

## Toolbar Attributes

- `theme` 主题，深色还是浅色
- `round` 圆角
- `size` 大小 `large` `middle` `small` `mini`
- `block` 是否为块级元素
- `background` 自定义背景色
- `text-color` 文字颜色
- `fixed` 是否固定定位	
- `fixed-style` 固定定位的样式，仅支持 `top` `right` `button` `left`，可参考 [Alert](https://fighting.tianyuhao.cn/components/alert.html#attributes)

## Toolbar Slots

- `default` 默认插槽，插入 `f-toolbar-item`

## Toolbar-item Attributes

后续更新

## Toolbar-item Slots

- `default` 默认插槽

## Toolbar-item Events

- `on-click` 点击之后的回调

**Comment by wmasfoe** [comment]
这个组件有人开发吗？

**Reply by Tyh2001** [reply]
https://fighting.tianyuhao.cn/components/toolbar.html

已经写好了，哈哈 @Rin-Nohara

**Reply by wmasfoe** [reply]
兄弟们效率太高了……
